xml/docker_building_images.xml

```diff
@@ -61,10 +61,16 @@ xml:id="docker.building.images" xml:lang="en">
   <title>Obtaining Base &slsa; Images</title>
 
   <para>
-   You can install pre-built images of &slsa; by using Zypper:
+   You can install pre-built images of &slsa; up to &slsa;&nbsp;12 SP2 by using Zypper:
   </para>
 
-<screen>sudo zypper in sles11sp4-docker-image suse-sles12sp4-image</screen>
+  <screen>sudo zypper in sles11sp4-docker-image</screen>
+
+  <para>
+   For newer &slsa; versions, use <literal>docker pull</literal>:
+  </para>
+
+   <screen>docker pull registry.suse.com/suse/sles12sp4</screen>
 
```

Comment:
The distinction between the "old" and "new" method is not obvious if it is made within a floating text. I suggest to use a variablelist:

------------------
How to obtain a base image depends on the &productname; version:

Up to slsa;&nbsp;12 SP2
  sudo zypper in sles11sp4-docker-image suse-sles12sp4-image

slsa;&nbsp;12 SP2 and better
  sudo zypper in sles11sp4-docker-image
--------------------
And do not forget using $prompt.sudo; instead of sudo

Reply:
fixed, see 743dfb5